packages/accounts-base/accounts_client.js

```diff
@@ -63,6 +66,45 @@ export class AccountsClient extends AccountsCommon {
   }
 
   /**
+   * @summary Register a new login function on client. Intended for OAuth package authors. You can call the login function by using `Accounts.callLoginFunction`.
+   * @locus Client
+   * @param {String} funcName The name of your login function. Used by `Accounts.callLoginFunction`. Should be the OAuth provider name accordingly.
+   * @param {Function} func The actual function you want to call. Just write it in the manner of `loginWithFoo`.
+   */
+  registerClientLoginFunction(funcName, func) {
+    if (this._loginFuncs[funcName]) {
+      throw new Error(`${funcName} has been defined already`);
+    }
+    this._loginFuncs[funcName] = func;
+  }
```

Comment:
Super nit picky, but could you add a blank line between the end/start of the new functions you've added? (to mirror the rest of the formatting in the file)